traefik-golang/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+  frontend:
+    image: traefik:2.2
+    command: --providers.docker --entrypoints.web.address=:80 --providers.docker.exposedbydefault=false
```

Comment:
ENV variables seems to be a better choice, they can be overridden, WDYT?

Reply:
env vars is the worse choice, they cannot be overridden after the start of Traefik.

The best choice is the CLI flags.

Reply:
Done !
And I think using a traefik.yml or toml wouldn't be that easy. I wanted to use the simplest possible solution.